sdcm/logcollector.py

```diff
@@ -806,7 +806,9 @@ class ScyllaLogCollector(LogCollector):
                     CommandLog(name='systemctl.status',
                                command='sudo systemctl status --all --full --no-pager'),
                     CommandLog(name='cassandra-rackdc.properties',
-                               command=f'cat {SCYLLA_PROPERTIES_PATH}')
+                               command=f'cat {SCYLLA_PROPERTIES_PATH}'),
+                    CommandLog(name='scripts_debug.log',
```

Comment:
Can we have a better name ?

Like boot_script_errors.log or init_scripts_errors.log ?

Reply:
renamed to setup_scripts_errors.log